frontend/src/pages/audits/edit/findings/add/add.html

```diff
@@ -60,6 +60,7 @@
                         v-model="search.title"
```

Comment:
Checkboxes should be on the left of the table with button to add on the table header to have a correct UX